docs/src/start/device-monitoring.md

```diff
@@ -12,63 +12,48 @@ Using these metrics, you can monitor the health of devices
 and can proactively initiate actions in case the device seems to malfunction.
 Additionally, the metrics can be used to help the customer troubleshoot when problems with the device are reported.
 
-%%te%% uses the open source component [`collectd`](https://collectd.org/) to collect the metrics from the device.
-%%te%% translates the collected metrics from their native format to the [%%te%% JSON](../understand/thin-edge-json.md) format
+%%te%% uses the open source component [collectd](https://collectd.org/) to collect the metrics from the device.
+%%te%% translates the collectd metrics from their native format to the [%%te%% JSON](../understand/thin-edge-json.md) format
```

Comment:
I don't think it was I typo. I would write either "collected metrics" or "`collectd` metrics".

Reply:
Yes there were a few cases that I read that I couldn't figure out where it was "collectd" and "collected"...however I think "collectd" is more accurate as it refers to "their native format" which is ambiguous if "collected" was used (imo).

Reply:
But yes using `collectd` will be better here to ensure people don't think it is a typo